### PR TITLE
Truth enforcers premium warbond

### LINKED ISSUE #68: [BUG]: Truth Enforcers Warbond
### Faulty File Path & Name

Various

### Request information

Add new warbond items

### What is expected?

Warbond present

### What is actually happening?

```json
Warbond missing (its not out yet dw)
```

### Any additional comments?

_I can do the PR_

### PR DESCRIPTION
**Pull request by Stonemercy**
Fixes #68

### CODE REVIEW

**Comment by Stonemercy** [comment]
image
image
image

File: items/armor/armor.json [hunk 1]
```diff
@@ -2258,5 +2258,95 @@
 		"speed": 100,
 		"stamina_regen": 100,
 		"passive": 0
+	},
+	"1010": {
```
Comment (chatterchats) [inline items/armor/armor.json:2262]:
```suggestion
	"3008695584": {
```

File: items/armor/armor.json [hunk 2]
```diff
@@ -2258,5 +2258,95 @@
...
+		"name": "UF-50 Bloodhound",
+		"description": "",
```
Comment (chatterchats) [inline items/armor/armor.json:2264]:
```suggestion
		"description": "Users are advised not to scratch at the lacquer, as the Red 20,000 pigment that gives this armor its signature sheen may be correlated with rapid DNA unspooling.",
```

File: items/armor/armor.json [hunk 3]
```diff
@@ -2258,5 +2258,95 @@
...
+		"type": 1,
+		"slot": 2,
+		"armor_rating": 100,
+		"speed": 500,
+		"stamina_regen": 100,
+		"passive": 13
+	},
+	"1011": {
```
Comment (chatterchats) [inline items/armor/armor.json:2272]:
```suggestion
	"238751935": {
```

File: items/armor/armor.json [hunk 4]
```diff
@@ -2258,5 +2258,95 @@
...
+		"name": "UF-50 Bloodhound",
+		"description": "",
```
Comment (chatterchats) [inline items/armor/armor.json:2274]:
```suggestion
		"description": "Users are advised not to scratch at the lacquer, as the Red 20,000 pigment that gives this armor its signature sheen may be correlated with rapid DNA unspooling.",
```

File: items/armor/armor.json [hunk 5]
```diff
@@ -2258,5 +2258,95 @@
...
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"1012": {
```
Comment (chatterchats) [inline items/armor/armor.json:2282]:
```suggestion
	"1126585234": {
```

File: items/armor/armor.json [hunk 6]
```diff
@@ -2258,5 +2258,95 @@
...
+		"name": "Pride of the Whistleblower",
+		"description": "Granted to the Truth Enforcers' Informant of the Month, this tasteful cape conjures images of dissident blood on asphalt.",
+		"type": 1,
+		"slot": 1,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"1015": {
```
Comment (chatterchats) [inline items/armor/armor.json:2292]:
```suggestion
	"3530620108": {
```

File: items/armor/armor.json [hunk 7]
```diff
@@ -2258,5 +2258,95 @@
...
+		"name": "UF-16 Inspector",
+		"description": "",
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"1016": {
```
Comment (chatterchats) [inline items/armor/armor.json:2302]:
```suggestion
	"3059832618": {
```

File: items/armor/armor.json [hunk 8]
```diff
@@ -2258,5 +2258,95 @@
...
+		"type": 1,
+		"slot": 1,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"1015": {
+		"name": "UF-16 Inspector",
+		"description": "",
```
Comment (chatterchats) [inline items/armor/armor.json:2294]:
```suggestion
		"description": "Made famous by the \"Truth Enforcers: Enforcers of Truth\" mystery novels, this armor calls to mind those paragons of patriotism who protect us from treasonous lies.",
```

File: items/armor/armor.json [hunk 9]
```diff
@@ -2258,5 +2258,95 @@
...
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"1016": {
+		"name": "UF-16 Inspector",
+		"description": "",
```
Comment (chatterchats) [inline items/armor/armor.json:2304]:
```suggestion
		"description": "Made famous by the \"Truth Enforcers: Enforcers of Truth\" mystery novels, this armor calls to mind those paragons of patriotism who protect us from treasonous lies.",
```

File: items/weapons/primary.json [hunk 10]
```diff
@@ -442,5 +442,33 @@
...
+		"recoil": 120,
+		"fire_rate": 80,
+		"fire_mode": [],
+		"traits": [
+			1,
+			2,
+			5
+		]
+	},
+	"1006": {
```
Comment (chatterchats) [inline items/weapons/primary.json:461]:
```suggestion
	"725205791": {
```

File: items/weapons/secondary.json [hunk 11]
```diff
@@ -139,5 +139,20 @@
 			5,
 			11
 		]
+	},
+	"1018": {
```
Comment (chatterchats) [inline items/weapons/secondary.json:143]:
```suggestion
	"1902682578": {
```

File: warbonds/truth_enforcers.json [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+            {
+                "item_id": 1008,
+                "medal_cost": 5
+            },
+            {
+                "item_id": 4219002488,
+                "medal_cost": 12
+            },
+            {
+                "item_id": 1010,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:47]:
```suggestion
                "item_id": 3008695584,
```

File: items/item_names.json [hunk 13]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1007": {
+		"name": "Inconspicuous Black Shuttle",
+		"mix_id": "pattern"
+	},
+	"1008": {
+		"name": "Pride of the Whistleblower",
+		"mix_id": "playercard"
+	},
+	"1010": {
```
Comment (chatterchats) [inline items/item_names.json:1486]:
```suggestion
	"3008695584": {
```

File: items/item_names.json [hunk 14]
```diff
@@ -1454,5 +1454,73 @@
...
+		"name": "UF-50 Bloodhound",
+		"mix_id": "body"
```
Comment (chatterchats) [inline items/item_names.json:1488]:
```suggestion
		"mix_id": "1927188023"
```

File: items/item_names.json [hunk 15]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1011": {
```
Comment (chatterchats) [inline items/item_names.json:1490]:
```suggestion
	"238751935": {
```

File: items/item_names.json [hunk 16]
```diff
@@ -1454,5 +1454,73 @@
...
+		"name": "UF-50 Bloodhound",
+		"mix_id": "helmet"
```
Comment (chatterchats) [inline items/item_names.json:1492]:
```suggestion
		"mix_id": "4286615047"
```

File: warbonds/truth_enforcers.json [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+                "medal_cost": 45
+            },
+            {
+                "item_id": 1011,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:51]:
```suggestion
                "item_id": 4286615047,
```

File: items/item_names.json [hunk 18]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1012": {
```
Comment (chatterchats) [inline items/item_names.json:1494]:
```suggestion
	"1126585234": {
```

File: items/item_names.json [hunk 19]
```diff
@@ -1454,5 +1454,73 @@
...
+		"name": "Pride of the Whistleblower",
+		"mix_id": "cape"
```
Comment (chatterchats) [inline items/item_names.json:1496]:
```suggestion
		"mix_id": "1126585234"
```

File: warbonds/truth_enforcers.json [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+                "medal_cost": 35
+            },
+            {
+                "item_id": 1012,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:55]:
```suggestion
                "item_id": 1126585234,
```

File: items/item_names.json [hunk 21]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1013": {
+		"name": "Proof of Faultless Virtue",
+		"mix_id": "playercard"
+	},
+	"1014": {
+		"name": "Fee of Thought",
+		"mix_id": "title"
+	},
+	"1015": {
```
Comment (chatterchats) [inline items/item_names.json:1506]:
```suggestion
	"3530620108": {
```

File: items/item_names.json [hunk 22]
```diff
@@ -1454,5 +1454,73 @@
...
+		"name": "UF-16 Inspector",
+		"mix_id": "helmet"
```
Comment (chatterchats) [inline items/item_names.json:1508]:
```suggestion
		"mix_id": "3285635529"
```

File: warbonds/truth_enforcers.json [hunk 23]
```diff
@@ -0,0 +1,97 @@
...
+            {
+                "item_id": 1013,
+                "medal_cost": 15
+            },
+            {
+                "item_id": 1014,
+                "medal_cost": 20
+            },
+            {
+                "item_id": 1015,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:72]:
```suggestion
                "item_id": 3530620108,
```

File: items/item_names.json [hunk 24]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1016": {
```
Comment (chatterchats) [inline items/item_names.json:1510]:
```suggestion
	"3059832618": {
```

File: items/item_names.json [hunk 25]
```diff
@@ -1454,5 +1454,73 @@
...
+		"name": "UF-16 Inspector",
+		"mix_id": "body"
```
Comment (chatterchats) [inline items/item_names.json:1512]:
```suggestion
		"mix_id": "4168334185"
```

File: warbonds/truth_enforcers.json [hunk 26]
```diff
@@ -0,0 +1,97 @@
...
+                "medal_cost": 45
+            },
+            {
+                "item_id": 1016,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:76]:
```suggestion
                "item_id": 3059832618,
```

File: items/item_names.json [hunk 27]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1017": {
```
Comment (chatterchats) [inline items/item_names.json:1514]:
```suggestion
	"3965478926": {
```

File: warbonds/truth_enforcers.json [hunk 28]
```diff
@@ -0,0 +1,97 @@
...
+                "medal_cost": 65
+            },
+            {
+                "item_id": 1017,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:80]:
```suggestion
                "item_id": 3965478926,
```

File: items/item_names.json [hunk 29]
```diff
@@ -1454,5 +1454,73 @@
...
+		"name": "Proof of Faultless Virtue",
+		"mix_id": "cape"
+	},
+	"1018": {
```
Comment (chatterchats) [inline items/item_names.json:1518]:
```suggestion
	"1902682578": {
```

File: warbonds/truth_enforcers.json [hunk 30]
```diff
@@ -0,0 +1,97 @@
...
+                "medal_cost": 30
+            },
+            {
+                "item_id": 1018,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:84]:
```suggestion
                "item_id": 1902682578,
```

File: warbonds/truth_enforcers.json [hunk 31]
```diff
@@ -0,0 +1,97 @@
...
+            {
+                "item_id": 1001,
+                "medal_cost": 50
+            },
+            {
+                "item_id": 4219002488,
+                "medal_cost": 7
+            },
+            {
+                "item_id": 1003,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:14]:
```suggestion
                "item_id": 122941246,
```

File: warbonds/truth_enforcers.json [hunk 32]
```diff
@@ -0,0 +1,97 @@
...
+                "item_id": 1005,
+                "medal_cost": 10
+            }
+        ]
+    },
+    "2": {
+        "medals_to_unlock": 90,
+        "items": [
+            {
+                "item_id": 1006,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:31]:
```suggestion
                "item_id": 725205791,
```

File: items/item_names.json [hunk 33]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1005": {
+		"name": "Inconspicuous Black Hellpod",
+		"mix_id": "pattern"
+	},
+	"1006": {
+		"name": "SMG-32 Reprimand",
+		"mix_id": "smg"
+	},
+	"1007": {
```
Comment (chatterchats) [inline items/item_names.json:1478]:
```suggestion
	"2484094371": {
```

File: warbonds/truth_enforcers.json [hunk 34]
```diff
@@ -0,0 +1,97 @@
...
+                "medal_cost": 40
+            },
+            {
+                "item_id": 1007,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:35]:
```suggestion
                "item_id": 2484094371,
```

File: items/item_names.json [hunk 35]
```diff
@@ -1454,5 +1454,73 @@
...
+	},
+	"1003": {
+		"name": "SG-20 Halt",
+		"mix_id": "shotgun"
+	},
+	"1004": {
+		"name": "Dead Sprint",
+		"mix_id": "booster"
+	},
+	"1005": {
```
Comment (chatterchats) [inline items/item_names.json:1470]:
```suggestion
	"2579792615": {
```

File: warbonds/truth_enforcers.json [hunk 36]
```diff
@@ -0,0 +1,97 @@
...
+            {
+                "item_id": 1003,
+                "medal_cost": 20
+            },
+            {
+                "item_id": 1004,
+                "medal_cost": 35
+            },
+            {
+                "item_id": 1005,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:22]:
```suggestion
                "item_id": 2579792615,
```

File: items/item_names.json [hunk 37]
```diff
@@ -1454,5 +1454,73 @@
 	"3236079222": {
 		"name": "Mustard Exosuit",
 		"mix_id": "3236079222"
+	},
+	"1001": {
```
Comment (chatterchats) [inline items/item_names.json:1458]:
```suggestion
	"3241211079": {
```

File: warbonds/truth_enforcers.json [hunk 38]
```diff
@@ -0,0 +1,97 @@
+{
+    "1": {
+        "medals_to_unlock": 0,
+        "items": [
+            {
+                "item_id": 1001,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:6]:
```suggestion
                "item_id": 3241211079,
```

**Review by chatterchats — CHANGES_REQUESTED** [review]
Made suggestions above for IDs and Descriptions. Please verify no missing syntax for each suggest, and prefferably batch merge. Will get the unfinished ones after that's done

File: items/boosters.json [hunk 39]
```diff
@@ -46,5 +46,9 @@
 	"3102833400": {
 		"name": "Firebomb Hellpods",
 		"description": "Lines all Hellpods with volatile incendiaries that detonate upon impact, igniting any units in vicinity of the drop site."
+	},
+	"1004": {
```
Comment (chatterchats) [inline items/boosters.json:50]:
```suggestion
	"2239116615": {
```

File: items/item_names.json [hunk 40]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"3241211079": {
+		"name": "Inconspicuous Black Exosuit",
+		"mix_id": "pattern"
+	},
+	"122941246": {
+		"name": "SG-20 Halt",
+		"mix_id": "122941246"
+	},
+	"1004": {
```
Comment (chatterchats) [inline items/item_names.json:1466]:
```suggestion
	"2239116615": {
```

File: items/item_names.json [hunk 41]
```diff
@@ -1454,5 +1454,85 @@
...
+		"name": "Dead Sprint",
+		"mix_id": "booster"
```
Comment (chatterchats) [inline items/item_names.json:1468]:
```suggestion
		"mix_id": "2239116615"
```

File: warbonds/truth_enforcers.json [hunk 42]
```diff
@@ -0,0 +1,97 @@
...
+            {
+                "item_id": 4219002488,
+                "medal_cost": 7
+            },
+            {
+                "item_id": 122941246,
+                "medal_cost": 20
+            },
+            {
+                "item_id": 1004,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:18]:
```suggestion
                "item_id": 2239116615,
```

File: warbonds/truth_enforcers.json [hunk 43]
```diff
@@ -0,0 +1,97 @@
...
+            {
+                "item_id": 725205791,
+                "medal_cost": 40
+            },
+            {
+                "item_id": 2484094371,
+                "medal_cost": 50
+            },
+            {
+                "item_id": 1008,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:39]:
```suggestion
                "item_id": 601062191,
```

File: items/item_names.json [hunk 44]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"725205791": {
+		"name": "SMG-32 Reprimand",
+		"mix_id": "725205791"
+	},
+	"2484094371": {
+		"name": "Inconspicuous Black Shuttle",
+		"mix_id": "pattern"
+	},
+	"1008": {
```
Comment (chatterchats) [inline items/item_names.json:1482]:
```suggestion
	"601062191": {
```

File: items/item_names.json [hunk 45]
```diff
@@ -1454,5 +1454,85 @@
...
+		"name": "Pride of the Whistleblower",
+		"mix_id": "playercard"
```
Comment (chatterchats) [inline items/item_names.json:1484]:
```suggestion
		"mix_id": "601062191"
```

File: warbonds/truth_enforcers.json [hunk 46]
```diff
@@ -0,0 +1,97 @@
...
+                "item_id": 1126585234,
+                "medal_cost": 20
+            }
+        ]
+    },
+    "3": {
+        "medals_to_unlock": 250,
+        "items": [
+            {
+                "item_id": 1013,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:64]:
```suggestion
                "item_id": 151318266,
```

File: items/item_names.json [hunk 47]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"238751935": {
+		"name": "UF-50 Bloodhound",
+		"mix_id": "4286615047"
+	},
+	"1126585234": {
+		"name": "Pride of the Whistleblower",
+		"mix_id": "1126585234"
+	},
+	"1013": {
```
Comment (chatterchats) [inline items/item_names.json:1498]:
```suggestion
	"151318266": {
```

File: items/item_names.json [hunk 48]
```diff
@@ -1454,5 +1454,85 @@
...
+		"name": "Proof of Faultless Virtue",
+		"mix_id": "playercard"
```
Comment (chatterchats) [inline items/item_names.json:1500]:
```suggestion
		"mix_id": "151318266"
```

File: warbonds/truth_enforcers.json [hunk 49]
```diff
@@ -0,0 +1,97 @@
...
+                "medal_cost": 15
+            },
+            {
+                "item_id": 1014,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:68]:
```suggestion
                "item_id": 1301492156,
```

File: items/item_names.json [hunk 50]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"1014": {
```
Comment (chatterchats) [inline items/item_names.json:1502]:
```suggestion
	"1301492156": {
```

File: warbonds/truth_enforcers.json [hunk 51]
```diff
@@ -0,0 +1,97 @@
...
+            {
+                "item_id": 3965478926,
+                "medal_cost": 30
+            },
+            {
+                "item_id": 1902682578,
+                "medal_cost": 45
+            },
+            {
+                "item_id": 1019,
```
Comment (chatterchats) [inline warbonds/truth_enforcers.json:88]:
```suggestion
                "item_id": 4693889,
```

File: items/item_names.json [hunk 52]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"3965478926": {
+		"name": "Proof of Faultless Virtue",
+		"mix_id": "cape"
+	},
+	"1902682578": {
+		"name": "PLAS-15 Loyalist",
+		"mix_id": "1902682578"
+	},
+	"1019": {
```
Comment (chatterchats) [inline items/item_names.json:1522]:
```suggestion
	"4693889": {
```

File: items/item_names.json [hunk 53]
```diff
@@ -1454,5 +1454,85 @@
...
+		"name": "At Ease",
+		"mix_id": "emote"
```
Comment (chatterchats) [inline items/item_names.json:1524]:
```suggestion
		"mix_id": "4693889"
```

File: items/item_names.json [hunk 54]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"2001": {
+		"name": "UF-84 Doubt Killer",
+		"mix_id": "body"
+	},
+	"2002": {
```
Comment (chatterchats) [inline items/item_names.json:1530]:
```suggestion
	"238751935": {
```

File: items/item_names.json [hunk 55]
```diff
@@ -1454,5 +1454,85 @@
...
+		"name": "UF-84 Doubt Killer",
+		"mix_id": "head"
```
Comment (chatterchats) [inline items/item_names.json:1532]:
```suggestion
		"mix_id": "4286615047"
```

File: items/item_names.json [hunk 56]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"1902682578": {
+		"name": "PLAS-15 Loyalist",
+		"mix_id": "1902682578"
+	},
+	"1019": {
+		"name": "At Ease",
+		"mix_id": "emote"
+	},
+	"2001": {
```
Comment (chatterchats) [inline items/item_names.json:1526]:
```suggestion
	"622973953": {
```

File: items/item_names.json [hunk 57]
```diff
@@ -1454,5 +1454,85 @@
...
+		"name": "UF-84 Doubt Killer",
+		"mix_id": "body"
```
Comment (chatterchats) [inline items/item_names.json:1528]:
```suggestion
		"mix_id": "940858144"
```

File: items/item_names.json [hunk 58]
```diff
@@ -1454,5 +1454,85 @@
...
+	},
+	"2002": {
+		"name": "UF-84 Doubt Killer",
+		"mix_id": "head"
+	},
+	"2003": {
```
Comment (chatterchats) [inline items/item_names.json:1534]:
```suggestion
	"66704491": {
```

File: items/item_names.json [hunk 59]
```diff
@@ -1454,5 +1454,85 @@
...
+		"name": "Cover of Darkness",
+		"mix_id": "cape"
```
Comment (chatterchats) [inline items/item_names.json:1536]:
```suggestion
		"mix_id": "66704491"
```

**Review by chatterchats — CHANGES_REQUESTED** [review]
Changes note

File: items/weapons/primary.json [hunk 60]
```diff
@@ -442,5 +442,33 @@
...
+	},
+	"122941246": {
+		"name": "SG-20 Halt",
+		"description": "A humane compliance weapon with two seperate magazines: one for armor-penetrating flechette rounds and one for stun rounds.",
+		"type": 2,
+		"damage": 385,
+		"capacity": 16,
+		"recoil": 120,
+		"fire_rate": 80,
+		"fire_mode": [],
```
Comment (chatterchats) [inline items/weapons/primary.json:454]:
```suggestion
		"fire_mode": [
		    0
		],
```

File: items/weapons/primary.json [hunk 61]
```diff
@@ -442,5 +442,33 @@
...
+	},
+	"725205791": {
+		"name": "SMG-32 Reprimand",
+		"description": "A heavy-duty submachine gun that fires large-caliber rounds, with a slower rate of fire to manage the high recoil. Ideal for use at close to medium range",
+		"type": 3,
+		"damage": 125,
+		"capacity": 25,
+		"recoil": 44,
+		"fire_rate": 490,
+		"fire_mode": [],
```
Comment (chatterchats) [inline items/weapons/primary.json:469]:
```suggestion
		"fire_mode": [
		    0,
		    1,
		    2
		],
```

**Review by chatterchats — CHANGES_REQUESTED** [review]
Firemodes for Primary

**Review by chatterchats — APPROVED** [review]

**Pull request merged**